src/main/steps/applicant1/dob/content.ts

```diff
@@ -1,40 +1,47 @@
 import { CaseDate } from '../../../app/case/case';
+import { ApplyingWith } from '../../../app/case/definition';
 import { TranslationFn } from '../../../app/controller/GetController';
 import { FormContent } from '../../../app/form/Form';
 import { covertToDateObject } from '../../../app/form/parser';
 import { areDateFieldsFilledIn, isDateInputInvalid, isFutureDate } from '../../../app/form/validation';
+import { CommonContent } from '../../../steps/common/common.content';
 
-export const en = (): Record<string, unknown> => ({
-  section: 'Primary applicant',
-  title: "What's your date of birth?",
-  hint: 'For example, 28 6 1997',
-  errors: {
-    applicant1DateOfBirth: {
-      required: 'Enter your date of birth',
-      invalidDate: 'Date of birth must be a real date',
-      incompleteDay: 'Your date of birth must include a day',
-      incompleteMonth: 'Your date of birth must include a month',
-      incompleteYear: 'Your date of birth must include a year',
-      invalidDateInFuture: 'Your date of birth must be in the past',
+export const en = ({ userCase }: CommonContent): Record<string, unknown> => {
+  const section = userCase?.applyingWith === ApplyingWith.ALONE ? 'Applicant' : 'First applicant';
+  return {
+    section,
+    title: "What's your date of birth?",
+    hint: 'For example, 27 3 2007',
+    errors: {
+      applicant1DateOfBirth: {
+        required: 'Enter your date of birth',
+        invalidDate: 'Date of birth must be a real date',
+        incompleteDay: 'Your date of birth must include a day',
+        incompleteMonth: 'Your date of birth must include a month',
+        incompleteYear: 'Your date of birth must include a year',
+        invalidDateInFuture: 'Your date of birth must be in the past',
+      },
     },
-  },
-});
-
-export const cy = (): Record<string, unknown> => ({
-  section: 'Primary applicant (in Welsh)',
-  title: "What's your date of birth? (in Welsh)",
-  hint: 'For example, 28 6 1997 (in Welsh)',
-  errors: {
-    applicant1DateOfBirth: {
-      required: 'Enter your date of birth (in Welsh)',
-      invalidDate: 'Date of birth must be a real date (in Welsh)',
-      incompleteDay: 'Your date of birth must include a day (in Welsh)',
-      incompleteMonth: 'Your date of birth must include a month (in Welsh)',
-      incompleteYear: 'Your date of birth must include a year (in Welsh)',
-      invalidDateInFuture: 'Your date of birth must be in the past (in Welsh)',
+  };
+};
+export const cy = ({ userCase }: CommonContent): Record<string, unknown> => {
+  const section = userCase?.applyingWith === ApplyingWith.ALONE ? 'Applicant (in Welsh)' : 'First applicant (in Welsh)';
```

Comment:
wondering if this logic can be moved to some util file, it would simplify the code as well as unit tests